packages/client/components/InvitationLink.tsx

```diff
@@ -1,16 +1,28 @@
 import React from 'react'
-import {createFragmentContainer} from 'react-relay'
+import {PreloadedQuery, usePreloadedQuery} from 'react-relay'
 import graphql from 'babel-plugin-relay/macro'
-import {InvitationLink_massInvitation} from '../__generated__/InvitationLink_massInvitation.graphql'
+import {InvitationLinkRootQuery} from '../__generated__/InvitationLinkRootQuery.graphql'
 import TeamInvitationMeetingAbstract from './TeamInvitationMeetingAbstract'
 import InvitationLinkDialog from './InvitationLinkDialog'
 
 interface Props {
-  massInvitation: InvitationLink_massInvitation
+  queryRef: PreloadedQuery<InvitationLinkRootQuery>
 }
 
 function InvitationLink(props: Props) {
-  const {massInvitation} = props
+  const {queryRef} = props
+  const data = usePreloadedQuery<InvitationLinkRootQuery>(
+    graphql`
+      query InvitationLinkRootQuery($token: ID!) {
```

Comment:
-1 The query should be renamed to the place where it lives, so in this case
```
query InvitationLinkQuery($token: ID!) {
```

Reply:
Since it's not on the Root page that needs to be modified, this should just be a naming convention issue, right? Is there a similar specification document like this?